lib/active_merchant/billing/gateways/braintree/braintree_common.rb

```diff
@@ -18,4 +23,12 @@ def scrub(transcript)
       gsub(%r((&?ccnumber=)\d*(&?)), '\1[FILTERED]\2').
       gsub(%r((&?cvv=)\d*(&?)), '\1[FILTERED]\2')
   end
+
+  def supports_network_tokenization?
+    response = authorize(1, ActiveMerchant::Billing::NetworkTokenizationCreditCard.test_credit_card)
+    response_code = response.params['braintree_transaction'].try(:[], 'processor_response_code')
+    raise(ActiveMerchant::InvalidResponseError, response.message) unless response_code
+
+    response.params['braintree_transaction']['processor_response_code'] != ERROR_CODES.fetch(:payment_instrument_not_supported)
```

Comment:
> it's in a module that is used in this gateway as well 

Braintree does not use `ssl_post`, it uses the braintree gem here https://github.com/braintree/braintree_ruby/blob/02db6f8d75830aa3338d61985183aab4eebfab9b/lib/braintree/http.rb#L32 instead. 

I think it's ok to let this raise via hash fetch if the Authorize.net implementation isn't consistent anymore.